src/Ztm.WebApi/TransactionConfirmationWatchingRuleStatus.cs

```diff
@@ -0,0 +1,9 @@
+namespace Ztm.WebApi
+{
+    public enum TransactionConfirmationWatchingRuleStatus
```

Comment:
Same as previous issue. We should rename it to `TransactionConfirmationWatcherRuleStatus`.

Reply:
Renamed to `RuleStatus`